lib/voice/VoiceConnection.js

```diff
@@ -107,6 +118,10 @@ class VoiceConnection extends EventEmitter {
     #nonce = 0;
     #nonceBuffer = null;
     #stablelib = null;
+    /**
+     * @type {String?}
+     */
+    #userID = null;
```

Comment:
Probably a matter of taste but:
```suggestion
    /** @type {String?} */
    #userID = null;
```